assets/js/src/core/modules/data-object/editor/shared-tab-manager/tab-definitions.tsx

```diff
@@ -32,3 +33,14 @@ export const TAB_VERSIONS: IEditorTab = {
   icon: <Icon value={ 'history' } />,
   isDetachable: true
 }
+
+export const TAB_PREVIEW: IEditorTab = {
+  key: 'preview',
+  label: 'preview.label',
+  children: <PreviewView />,
+  icon: <Icon value={ 'preview' } />,
+  isDetachable: true,
```

Comment:
Hm... the preview detaching mechanism currrently does not work - I do not get a detached preview tab. If the preview is detached we would need to refresh it on each data object draft save (at the end).